README.md

```diff
@@ -87,18 +87,17 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 | Hosted  |                      |
 | Group   |                      |
 
-NOTE: This is an early version of P2 Proxy and does not fully support the following as of yet:
+NOTE: p2 Proxy does not fully support the following as of yet:
 
-* [Composite Repositories](https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fp2_composite_repositories.htm)
-  * This is implemented but likely will fail if the composite files reference absolute paths rather than relative paths
-* Old style (non-P2) update sites. The code that accomplished this can be found [here](https://github.com/sonatype/nexus-public/blob/nexus-2.x/plugins/p2/nexus-p2-repository-plugin/src/main/java/org/sonatype/nexus/plugins/p2/repository/proxy/P2ProxyRepositoryImpl.java)
-  * These are fairly old, and have not been added at all
+* Old style (non-p2) update sites. These are fairly old, and have not been added at all.
 
 If you'd like it to support the aforementioned sites, please file an issue, or better yet, submit a PR :)
 
 ## Installing the plugin
 
-There are a range of options for installing the P2 plugin. You'll need to build it first, and
+:bangbang: `After NXRM version 3.21, last released plugin's version is available directly in product and there is no need to install it separately` :bangbang:
```

Comment:
3.21 looks like should be with + sign

Reply:
IMO, + sign is redundant with "After".  This was already adjusted to this way at my request.  Alternatively, you could put "In NXRM versions 3.21+, the...".
I read the + as "and beyond" so saying "After NXRM version 3.21 and beyond"...maybe you can see my concern.